pom.xml

```diff
@@ -35,8 +35,8 @@
   </developers>
 
   <properties>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Spring Framework 5 requires JRE 8 so I believe this is safe to change.

Reply:
Note that this is not only SB, the web package can be used with old spring web apps.

Reply:
@pavolloffay Spring Framework 5 + required JDK 8+ see [here](https://github.com/spring-projects/spring-framework/wiki/Upgrading-to-Spring-Framework-5.x#upgrading-to-version-50)